packages/fast-components-react-msft/src/caption/README.md

```diff
@@ -0,0 +1,8 @@
+# Caption
+Caption component can help provide context to images, figures or tables.
+
+## Usage
+Caption should be used sparingly. Due to the small fontsize of captions, legibility may suffer, so it should not be used for long descriptions or as a paragraph. It should add informational value without being a distraction. Captions are also used for footnotes or as secondary text to provide additional information.
+
+## Style guidance
+There are two levels to caption. Level 1, sets the caption to font size of 10px, and level 2, sets the font size of 11px.
```

Comment:
I don't think we want to be this specific on the sizes, can we just say "The default size is level 1, with an additional larger size at level 2."